packages/fxa-auth-server/lib/senders/emails/templates/verify/en.ftl

```diff
@@ -1,5 +1,5 @@
-verify-title = Activate the { -brand-firefox } family of products
+verify-title-2 = Open the internet with { -brand-firefox }
```

Comment:
```suggestion
# Note for translation: In this context, open up is synonymous with "Discover". The metaphor is that when they open their Firefox browser, it helps them discover an amazing internet. If this translation does not make sense in your language, feel free to use the word "discover."
verify-title-2 = Open up the internet with { -brand-firefox }
```
This jogged a memory of a comment from content team in the MR1 2022 Onboarding figma. In it the text was "Open up" so perhaps it could be consistent with that? It also had the above comment pertinent to localizers. Would be great if we could have this comment in there to help localizers since "open" or "open up" could be translated in a few different ways, and this would help with keeping a consistent localization.